src/siembol/Chart.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
+- name: storm
+  repository: https://g-research.github.io/charts
+  version: 1.0.9
+- name: kafka
+  repository: https://charts.bitnami.com/bitnami
+  version: 12.16.2
+- name: hdfs
```

Comment:
this is question wheter we need this. hdfs is using in siembol for storing enrichment tables and we are planning to implement download the tables from http server.

Reply:
Yes we are using it in internal deployment but if we move to k8 with storm we will move tables to http server. I have created an issue in siembol repo for this task https://github.com/G-Research/siembol/issues/66

Reply:
'k, I'll leave this dependency in until we do #66.